Automattic/wp-calypso
# Add "Getting Started with Jetpack Backup" video card to Backup dashboard

#### Proposed Changes

We do want to serve the materials to users to help them out in their first steps in Jetpack Backup. Thus, we plan to add the dismissable card to the Backup dashboard informing them about the YT video that explains how to get started with the Backup feature.

#### Testing Instructions

1. Start a local development environment with ` yarn start-jetpack-cloud-p` (jetpack cloud) and `yarn start` (calypso)
2. Visit the page: http://jetpack.cloud.localhost:3001/backup/ (and select your test site)
3. Verify that the "Getting Started" card is displayed under the latest backup card:

<img width="812" alt="image" src="https://user-images.githubusercontent.com/8419292/173840849-2782f7f1-8203-41bb-8f2f-cc82cd38254e.png">

4. Change viewport to mobile.
5. Verify that "Getting Started" card is responsive:

<img width="529" alt="image" src="https://user-images.githubusercontent.com/8419292/173840999-df1c1995-3882-449f-9c9e-14453c781181.png">

6. Click "Watch a video" button - you should land at the following URL: https://www.youtube.com/watch?v=pS5-i5KjKTw
7. Visit the page: http://calypso.localhost:3000/backup/ (and select your test site)
8. Verify that the "Getting Started" card is displayed under the latest backup card:
<img width="747" alt="image" src="https://user-images.githubusercontent.com/8419292/174305339-f5950a1d-ea4f-4aa2-bb25-d861d267a2c3.png">


10. Click "X" on the "Getting Started" card
11. The "Getting Started" card should disappear from your dashboard
12. Reload the page
13. Once dismissed, the "Getting Started" card should not appear anymore
